source/Attenuator/Wireless.h

```diff
@@ -332,19 +346,25 @@ bool startWiFi() {
   // Assign an event handler to deal with changes in WiFi status.
   WiFi.onEvent(OnWiFiEvent);
 
-  if(b_wifi_enabled) {
-    // When external WiFi is desired, enable simultaneous SoftAP + Station mode.
-    WiFi.mode(WIFI_MODE_APSTA);
+  // Attempt connection to an external (preferred) WiFi as a client.
+  b_ext_wifi_started = startExternalWifi();
+
+  if(!b_wifi_enabled || !b_ext_wifi_started) {
+    #if defined(DEBUG_WIRELESS_SETUP)
+      Serial.println(F("External WiFi not available, switching to SoftAP mode..."));
+    #endif
+
+    // When external WiFi is unavailable, switch to only SoftAP mode.
+    WiFi.mode(WIFI_MODE_AP);
     delay(250);
   }
 
   // Start the built-in access point (softAP) with the preferred credentials.
+  // This should ALWAYS be available for direct connections to the device.
   if(!b_ap_started) {
     b_ap_started = startAccesPoint();
```

Comment:
Just a minor typo; function name should be `startAccessPoint()`.

Reply:
Damn, never noticed that. At least it's consistently misspelled for compilation :)